.werft/jobs/build/prepare.ts

```diff
@@ -110,7 +110,9 @@ function createVM(werft: Werft, config: JobConfig) {
     }
 
     werft.log(prepareSlices.BOOT_VM, "Creating  VM");
-    VM.startVM({ name: config.previewEnvironment.destname });
+    let cpu = config.withLargeVM ? 12 : 6;
```

Comment:
```suggestion
    const cpu = config.withLargeVM ? 12 : 6;
```

I think we can make both `cpu` and `memory` const because we don't modify them in this scope